install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
+openshift_prometheus_image_alertbuffer="ilackarms/message-buffer"
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+[openshift-prometheus--additional-deploy]
+==== Additional Methods for Deploying Prometheus
+
+*Deploy Using Node-Selector*
+
+Label the node on which you want to deploy Prometheus:
+----
+# oadm label node/$NODE ${KEY}=${VALUE}
+----
+
+Deploy Prometheus with Ansible and container resources:
+----
+# Inventory file
+openshift_prometheus_state=present
+openshift_prometheus_namespace=prometheus
+openshift_prometheus_replicas=1
+
+# Set node selector for prometheus
+openshift_prometheus_node_selector={"${KEY}":"${VALUE}"}
+
+# images
+openshift_prometheus_image_proxy="openshift/oauth-proxy:v1.0.0"
+openshift_prometheus_image_prometheus="openshift/prometheus:v2.0.0-dev"
+openshift_prometheus_image_alertmanager="openshift/prometheus-alertmanager:dev"
+openshift_prometheus_image_alertbuffer="ilackarms/message-buffer"
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+*Deploy Using a Non-default Namespace*
+
+Identify your namespace:
+----
+# Inventory file
+openshift_prometheus_state=present
+openshift_prometheus_replicas=1
+openshift_prometheus_node_selector={"region":"infra"}
+
+# Set non-default openshift_prometheus_namespace
+openshift_prometheus_namespace=${USER_PROJECT}
+
+# images
+openshift_prometheus_image_proxy="openshift/oauth-proxy:v1.0.0"
+openshift_prometheus_image_prometheus="openshift/prometheus:v2.0.0-dev"
+openshift_prometheus_image_alertmanager="openshift/prometheus-alertmanager:dev"
+openshift_prometheus_image_alertbuffer="ilackarms/message-buffer"
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+[openshift-prometheus-web]
+==== Accessing the Prometheus Web UI
+
+The Prometheus server automatically exposes a Web UI at `localhost:9090`. You
+can access the Prometheus Web UI with the `view` role.
+
+[openshift-prometheus-config]
+==== Configuring Prometheus for OpenShift
+
+Example Prometheus rules file:
+----
+# additional prometheus rules file
+openshift_prometheus_additional_rules_file: null
+
+# All the required exports
+openshift_prometheus_pv_exports:
+-  prometheus
+-  prometheus-alertmanager
+-  prometheus-alertbuffer
+# PV template files and their created object names
+  openshift_prometheus_pv_data:
+  -  pv_name: prometheus
+     pv_template: prom-pv-server.yml
+     pv_label: Prometheus Server PV
+  -  pv_name: prometheus-alertmanager
+     pv_template: prom-pv-alertmanager.yml
+     pv_label: Prometheus Alertmanager PV
+  -  pv_name: prometheus-alertbuffer
+     pv_template: prom-pv-alertbuffer.yml
+     pv_label: Prometheus Alert Buffer PV
+
+# Hostname/IP of the NFS server. Currently defaults to first master
```

Comment:
There are two choices
1. attach pv to prometheus
2. does't need to attach pv to prometheus

and we not only support NFS PVs,  do we need to tell customer how to attach pv？